### PR TITLE
Add docs and fix gradle

### LINKED ISSUE #3: Update Readme.md
Please add "How to run", "How to play" and "Features" sections.
Each one can be in separate PR

### PR DESCRIPTION
**Pull request by fadeltd**
Resolve #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,174 @@
...
+1. The game requires two players, X and O.
+2. The game board is a set 3x3 grid in which players will place their symbol to claim that segment.
+3. X typically players first, then players alternate turns.
+4. The goal is to claim three segments of the grid in a row, either horizontally, vertically, or diagonally.
+No additional sides can be added to the grid.
+5. The game is over either when one player achieves three segments in a row, or when the grid is filled without anyone achieving three segments in a row.
+
+In this game, after you select `user` (human player) to play the game, you can type in the coordinates to play the game. You'll be receiving this input command.
+
+```java
```
Comment (mdolata) [inline README.md]: it is not java code

**Review by mdolata — COMMENTED** [review]
I really appreciate your work, but is it ```java needed when it is not java code?

File: README.md (hunk 1 above)
Comment (fadeltd) [reply README.md]: should I change it to `sh`, since the input is in console?

Comment (mdolata) [reply README.md]: sure. good idea

**Pull request merged**